src/client/Microsoft.Identity.Client/PlatformsCommon/Polyfils/CertificateRequest.cs

```diff
@@ -0,0 +1,376 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.ObjectModel;
+using System.Formats.Asn1;
```

Comment:
This comes either from the direct package reference on net fwk, or from the implicit reference on net8